dbt_schema_builder/builder.py

```diff
@@ -185,27 +179,59 @@ def get_app_schema_configs(self):
         with open(schema_config_file_path, "r") as f:
             config = yaml.safe_load(f)
 
-        validate_schema_config(config)
+        self.validate_schema_config(config)
 
         return config
 
-    def get_project_dirs(self):
-        """
-        Find the dbt project directory based on the command line inputs.
-        """
-        source_project_path = os.getcwd()
-        destination_project_path = os.path.join(
-            source_project_path, self.args.destination_project
-        )
-
-        for project_path in [source_project_path, destination_project_path]:
-            if not os.path.exists(os.path.join(project_path, "dbt_project.yml")):
-                raise Exception(
-                    "fatal: {} is not a dbt project. Does not exist or is missing a "
-                    "dbt_project.yml file.".format(project_path)
-                )
-
-        return source_project_path, destination_project_path
+    @staticmethod
+    def validate_schema_config(config):
+        """
+        Read through an app-schema config dict, making sure that it meets certain
+        expectations before proceeding. If not, raise an exception to prevent
+        invalid schemas from being built
+        """
+        valid_keys = ['EXCLUDE', 'INCLUDE', 'SOFT_DELETE']
+        for _, app_config in config.items():
+            for schema, schema_config in app_config.items():
+                # This represents the case in which an application schema does
+                # not have any special logic concerning which tables to include
+                # or exclude
+                if not schema_config:
+                    continue
+                keys = schema_config.keys()
+                if 'EXCLUDE' in keys and 'INCLUDE' in keys:
+                    raise InvalidConfigurationException(
+                        "{} has both an EXCLUDE and INCUDE section".format(
+                            schema
+                        )
+                    )
+                if 'EXCLUDE' not in keys and 'INCLUDE' not in keys and 'SOFT_DELETE' not in keys:
```

Comment:
Actually, I don't know if I agree with this logic. It's possible we could say "include everything but there is a soft delete key". This would mean that the config doesn't have either an `INCLUDE` or an `EXCLUDE` key

Reply:
The issue I ran into here is that there was a key that was not INCLUDE or EXCLUDE (SOFT_DELETE) and it was failing. Since we're checking valid keys below I should just get rid of this if statement since the original assumption that if there were any keys at all we were dealing with an include/exclude situation is no longer true. I'll take care of that.